apps/extension/server/package.json

```diff
@@ -12,15 +11,14 @@
     "url": "https://github.com/Microsoft/vscode-extension-samples"
   },
   "dependencies": {
+    "@xstate/tools-shared": "1.0.0",
     "vscode-languageserver": "^7.0.0",
     "vscode-languageserver-textdocument": "^1.0.1",
-    "xstate": "^4.29.0",
-    "xstate-tools-shared": "*"
+    "xstate": "^4.29.0"
   },
   "devDependencies": {},
   "scripts": {
     "lint": "tsc",
-    "build:dev": "esbuild --bundle src/server.ts --outfile=dist/index.js --external:vscode --platform=\"node\"",
-    "build:prod": "ncc build src/server.ts -t -o dist"
+    "build": "esbuild --bundle src/server.ts --outfile=dist/index.js --external:vscode --external:prettier --platform=\"node\" --format=cjs"
```

Comment:
q: isn't this whole package part of the extension "package"? do we need multiple package.json files to describe a single extension? is this correctly setup right now with `--bundle` etc?